examples/search-chat-separate-routes/src/components/Layout/PageLayout.tsx

```diff
@@ -1,17 +1,22 @@
-import { PropsWithChildren } from "react";
+import { PropsWithChildren, useState } from "react";
 import Header from "./Header";
+import { EmbedConfigContext, defaultConfig } from "../../EmbedConfigContext";
 
-const PageLayout = ({ children }: PropsWithChildren) => (
-  <div className="w-full h-full flex flex-col items-center">
-    <div className="w-fixed w-full flex-shrink flex-grow-0 px-4 w-fixed w-full flex-shrink flex-grow-0 px-4 border-b border-gray-20 shadow-md">
-      <div className="sticky top-0 p-4 w-full h-full z-50">
-        <Header />
+const PageLayout = ({ children }: PropsWithChildren) => {
+  const [config, setConfig] = useState(defaultConfig)
```

Comment:
Is that because of params like `backend` that don't really get updated even if options are changed?

Reply:
Yeah, I tried updating backend but it didn't get reflected.

Reply:
Gotcha, in that case a reload + session storage would work. 
Another option is to delete the frame container by utilising a key which will remount the container div, for eg:
```
<Container key={join all keys that are initially reflected in query params} > </Container>

```

Reply:
> Another option is to delete the frame container by utilising a key which will remount the container div

Yeah, this would work. Using refresh mechanism for now.